### PR TITLE
Emit warning when asked to build a virtual proj

### PR DESCRIPTION
**Pull request by nazq**

### CODE REVIEW

File: rye/src/cli/build.rs [hunk 1]
```diff
@@ -69,6 +69,10 @@ pub fn execute(cmd: Args) -> Result<(), Error> {
     for project in projects {
         // skip over virtual packages on build
         if project.is_virtual() {
+            warn!(
+                "skipping virtual package {}",
+                style(project.normalized_name()?).cyan()
+            );
```
Comment (charliermarsh) [inline rye/src/cli/build.rs:75]: I wonder if we should only do this when _all_ projects are virtual?

Comment (nazq) [reply rye/src/cli/build.rs:75]: Great suggestion

Comment (nazq) [reply rye/src/cli/build.rs:83]: pushed an update

**Review by charliermarsh — APPROVED** [review]

**Comment by charliermarsh** [comment]
Thanks!

**Pull request merged**